subsys/bluetooth/crypto/CMakeLists.txt

```diff
@@ -2,6 +2,4 @@
 
 zephyr_library()
 
-zephyr_include_directories(include)
```

Comment:
Since we are defining a library, we need to define the public header location. We should keep the include-folder, and change this line to:

```c
zephyr_include_directories(PUBLIC include)
```

This is the correct way to make anything that links to this library gain access to the public headers. Without this, there is no point to defining a library. If we don't want a library, we should just use `target_sources` instead.

Reply:
@tejlmand can you weigh in on this? should we replace `zephyr_library()` with `add_target_sources()` or similar?

Reply:
@tejlmand note that I intend to send another PR removing the following lines from `subsys/bluetooth/CMakeLists.txt`:

```
 target_include_directories(subsys__bluetooth INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}) 
 zephyr_include_directories(${ZEPHYR_BASE}/subsys/bluetooth) 
```

Because we do not want `subsys/bluetooth` to be in the global include path.

Reply:
I've discussed this with @tejlmand and he told me that having `zephyr_library()` has nothing to do with exposing the same root folder in the global include namespace, so this change is acked by him.

Reply:
Like I said before, I will clean up the Bluetooth build system files and inclusion paths.